### DISCUSSION TITLE
表编号加粗，但是图编号不加粗

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#126] [Q&A]
表编号加粗，但是图编号不加粗。这一现象是正常的吗？caption都没有加粗。
![Screenshot 2022-04-19 171406](https://user-images.githubusercontent.com/45897471/163971364-287221dd-2f5a-46eb-b1e1-da91843e7eab.jpg)
![Screenshot 2022-04-19 171351](https://user-images.githubusercontent.com/45897471/163971372-27f1934b-7317-40eb-aef8-aaa6bb6817b0.jpg)

**Comment by tanukihee** [comment]
本科规定如此，硕博没有明确规定，可以自行修改，修改方法请阅读 caption 宏包文档

**Reply by earthmoon** [reply]
好的，谢谢

**Reply by xkwxdyy** [reply]
> 本科规定如此，硕博没有明确规定，可以自行修改，修改方法请阅读 caption 宏包文档

奇怪，这个规范说的很模糊，他没说是只有标题还是和序一起的字体
